client/vite.config.js

```diff
@@ -0,0 +1,39 @@
+import { defineConfig } from 'vite'
+import fs from 'fs/promises';
+
+/*
+  This override allows us to use .js files instead of exclusively .jsx.
+  We should remove as soon as video updates can be prioritized.
+*/
```

Comment:
By default, Vite wants us to use all `.jsx` files. In order to avoid a lot of video inconsistencies, we are using this override that Daniel recommended; so when and if we make video edits, we can address it then!

Reply:
Ohhh gotcha. Hmmm could you track in a ticket so we don't forget? Maybe something like "Lift-off Video Updates".

Reply:
done! https://airtable.com/appAhlVdr0ykrd7Tj/tblhWUuHMtcyJpZxp/viwTXi2TpdTurZhlY/rec0q6PJyclDOR9rs?blocks=hide